mission-vision-values.md

```diff
@@ -0,0 +1,99 @@
+# OTel Mission, Vision, and Values
+
+## Mission (our overall north star as a community)
+
+OpenTelemetry's Mission: **to enable effective observability through
+high-quality, turnkey, open-source telemetry.**
+
+## Vision (the world we imagine for OTel end-users)
+
+Excellent observability requires high-quality telemetry, as well as the
+performant, consistent instrumentation that makes it possible.
+
+Telemetry in this context is the firehose of raw observational data streaming
+out of software applications, and while “high-quality telemetry” may be a
+requirement for excellent observability, it’s still unreasonable and
+unrealistic to expect software developers to add or maintain the necessary
+instrumentation on their own. That is a massive undertaking, and in practice
+it’s not “just code” – there is also necessary alignment around protocols and
+semantic conventions for tags, attributes, and other metadata to consider.
+
+So how do we get high-quality, turnkey telemetry without a massive,
+unsustainable engineering effort? This is where OpenTelemetry comes in. To
+achieve our vision, OpenTelemetry sees five key opportunities, listed here:
+
+**Opportunity I: Telemetry should be easy.** With OpenTelemetry we want
+high-quality telemetry to be easy, especially for end-users. That means that
+OpenTelemetry should have fast time-to-value, set reasonable defaults yet allow
+for customization, and pair with excellent documentation and a top-tier overall
+developer experience.
+
+**Opportunity II: Telemetry should be universal.** Telemetry protocols and
+conventions should be unified across languages and signal types (tracing,
+metrics, logging, etc), not divergent or siloed. This means that OpenTelemetry
+aspires to find technical solutions that work consistently, both locally and
+globally.
+
+**Opportunity III: Telemetry should be vendor-neutral.** For decades, monitoring
+and observability vendors have created differentiation – and lock-in – through
+agents that automatically gather useful telemetry from a wide integration
+surface. While convenient, this lock-in is limiting for customers and inhibits
```

Comment:
>While convenient

I think this sentence works better without these 2 words.

Reply:
Agree with tigran here. no need to soften this sentence

Reply:
great point, removed!